pkgs/applications/audio/feishin/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchurl
+, appimageTools
+, mpv
+}:
+
+let
+  pname = "feishin";
+  version = "0.0.1-alpha5";
+  src = fetchurl {
+    url = "https://github.com/jeffvli/feishin/releases/download/v${version}/Feishin-${version}-linux-x86_64.AppImage";
+    sha256 = "sha256-YLs3aL8VAM15oHNm+dOHvSrqByBJ4fiUqDOt0AaqscQ=";
+  };
+  appimageContents = appimageTools.extractType2 { inherit pname version src; };
+in
+appimageTools.wrapType2 {
+  inherit pname version src;
+
+  extraInstallCommands = ''
+    mv $out/bin/${pname}-${version} $out/bin/${pname}
+
+    install -m 444 -D ${appimageContents}/${pname}.desktop -t $out/share/applications
+    substituteInPlace $out/share/applications/${pname}.desktop \
+      --replace 'Exec=AppRun --no-sandbox %U' 'Exec=${pname}'
+    cp -r ${appimageContents}/usr/share/icons $out/share
+  '';
+
+  extraPkgs = pkgs: [
```

Comment:
```suggestion
  extraPkgs = [
```

Reply:
Getting following error when trying to remove this
```
error: attempt to call something which is not a function but a list

       at /home/onny/projects/nixpkgs/pkgs/build-support/appimage/default.nix:51:49:

           50|       targetPkgs = pkgs: [ appimage-exec ]
           51|         ++ defaultFhsEnvArgs.targetPkgs pkgs ++ extraPkgs pkgs;
             |                                                 ^
           52|
(use '--show-trace' to show detailed location information)
```